docs/pages/tutorial/eas/using-github.mdx

```diff
@@ -0,0 +1,151 @@
+- Click the **Get started** button which opens a popup to authorize the Expo GitHub app. Click **Install and Authorize**.
+- Once the app is installed on our GitHub account, we need to link it to our Expo account. In the next popup, click **Link installation**.
+- Once the account is linked, it will show under **GitHub accounts**.
+
+<ImageSpotlight
+  alt="Linked GitHub account shows in Expo dashboard"
+  src="/static/images/tutorial/eas/github-01.png"
+  style={{ maxWidth: 800 }}
+/>
+
+</Step>
+
+<Step label="2">
+
+## Connect the GitHub repository
+
+To enable triggering builds from a GitHub repository, we need to connect it to our project in the Expo dashboard:
+
+- Open the project from the navigation menu under **Project settings**, and click **GitHub**.
+- Under **Connect a GitHub repository**, we'll see a list of our GitHub repos. We need to connect the right one. In the example, we're searching for our repo **sticker-smash.**
+
+<ImageSpotlight
+  alt="List of GitHub repositories to connect to the project in Expo dashboard"
+  src="/static/images/tutorial/eas/github-02.png"
+  style={{ maxWidth: 800 }}
+/>
+
+- Click **Connect** for the project repository.
+
+</Step>
+
+<Step label="3">
+
+## Configure repository settings in the Expo dashboard
+
+The Expo GitHub app needs to know where to find the source code of our project. By default, it selects the root directory using `/`. In our example project, the source code is also available in the root repository.
+
+<ImageSpotlight
+  alt="Default settings for configuring a GitHub repository in Expo dashboard"
+  src="/static/images/tutorial/eas/github-03.png"
+  style={{ maxWidth: 800 }}
+/>
+
+</Step>
+
+<Step label="4">
+
+## Trigger a build using a GitHub PR label
+
+The Expo GitHub app provides us [multiple options](/build/building-from-github/#trigger-a-build-from-github) to trigger a build, such as:
+
+- Manually from the Builds page for a specific platform
+- Automatically when new code is pushed to the repository
+- Automatically using GitHub PR labels
+
+To automatically trigger a build using a GitHub PR label, we're going to utilize the third option from the list above:
+
+- We need to specify the build image that we will be using. Open **eas.json**, and under the `development` profile, add [`android.image`](/eas/json/#image) and [`ios.image`](/eas/json/#image-1) properties and set their value to [`latest`](/build-reference/infrastructure/#configuring-build-environment).
+
+```json eas.json
+{
+  "build": {
+    "development": {
+      /* @hide ... */ /* @end */
+      /* @info */
+      "android": {
+        "image": "latest"
+      },
+      "ios": {
+        "image": "latest"
+      }
+      /* @end */
+    }
+  }
+  /* @hide ... */ /* @end */
+}
+```
+
+- Next, modify our app's JavaScript code. We create a git branch called `dev` and submit a PR to our example repository from that branch.
```

Comment:
```suggestion
- Next, create a new branch called `dev`, make a change in your app's JavaScript code, commit it, push the branch and create a PR from that branch.
```

Reply:
Thanks for the suggestion! I slightly modified it to keep the second person voice intact here (hope you won't mind).